test/Microsoft.Health.Fhir.R4.Tests.E2E/Rest/VersionSpecificTests.cs

```diff
@@ -25,6 +25,16 @@ public async Task GivenR4Server_WhenCapabilityStatementIsRetrieved_ThenCorrectVe
             await TestCapabilityStatementFhirVersion("4.0.0");
         }
 
+        [Fact]
+        public async Task GivenAnObservationDefinition_WhenCreating_ThenTheCorrectResponseShouldBeReturned()
+        {
+            var resource = Samples.GetJsonSample<ObservationDefinition>("ObservationDefinition-example");
```

Comment:
nit: maybe add the test in SearchParameterDefinitionBuilderTests since this logic is not version specific?

Reply:
Good point, I'll add a test there too